knowledge_base/Notifications.md

```diff
@@ -13,16 +15,16 @@
 - [Proposed Architecture](#proposed-architecture)
```

Comment:
We should remove the proposed / new architecture section -- basically anything besides current setup, again due to #6730.